docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -52,7 +64,214 @@ to delete this directory and start the node up again. If you are using the defau
 
 :::
 
-## Changing the chain ID
+## Starting a local node (manual configuration)
+
+Manually configuring and starting the Gno blockchain node is a bit more involved than simply initializing it "lazily",
+and involves the following steps:
+
+- generating the node secrets, and configuration
+- generating the `genesis.json`, and populating it
+- starting the node with the generated data
+
+### 1. Generate the node directory
+
+You can generate the default node directory using the following command:
+
+```shell
+gnoland init
+```
+
+This will initialize the following directory structure:
+
+```shell
+.
+└── gnoland-data/
+    ├── secrets/
+    │   ├── priv_validator_state.json
+    │   ├── node_key.json
+    │   └── priv_validator_key.json
+    └── config/
+       └── config.toml
+```
+
+A couple of things to note:
+
+- `gnoland init` initializes a default configuration
+- `gnoland init` initializes new node secrets (validator key, node p2p key)
+
+Essentially, `gnoland init` is simply a combination of `gnoland secrets generate` and `gnoland config generate`, with
+the default options enabled.
+
+#### Changing the node configuration
+
+To change the configuration params, such as for example the node's listen address, you can utilize the following
+command:
+
+```shell
+gnoland config set rpc.laddr tcp://0.0.0.0:26657
+```
+
+This will update the RPC listen address to `0.0.0.0:26657`. You can verify the configuration was updated by running:
+
+```go
+gnoland config get rpc.laddr
+```
+
+### 2. Generate the `genesis.json`
+
+:::info Where's the `genesis.json`?
```

Comment:
Nit: replace with "Where is `genesis.json`"

Reply:
Why?

Reply:
Sounds better. It's a nitpick, something based on a feel. Resolve this if you do not want to implement my suggestion.